components/Card.js

```diff
@@ -1,9 +1,13 @@
-import Image from 'next/image';
-import PropTypes from 'prop-types';
+import { useContextTheme } from 'components/ThemeContext';
+
+import Button from '../components/Button';
+
 import React from 'react';
 import clsx from 'clsx';
-import { useContextTheme } from 'components/ThemeContext';
 
+import PropTypes from 'prop-types';
+
+import Image from 'next/image';
```

Comment:
Hey @kaushik3pg, Just a small suggestion why keep the imports separated by a blank line.